README.md

```diff
@@ -77,6 +80,8 @@ A list of UI components built with Material-UI Design.
 
 A list of mature apps built with Material-UI Design.
 
+- [Material-ui Medium Blog](https://github.com/sabesansathananthan/material-ui-medium-blog)
+- [React Github Repo Cards](https://github.com/sabesansathananthan/react-github-repo-cards)
```

Comment:
This goes under the components section
and please add a description after '-' sign

eg: [bethesda.net](https://bethesda.net/) - The official site for Bethesda, publisher of Fallout, DOOM, Dishonored, Skyrim, Wolfenstein, The Elder Scrolls, more.

Reply:
Sure @nadunindunil ayya.